src/main/java/com/glucoseguardian/webbackend/application/rest/NotificaRest.java

```diff
@@ -0,0 +1,174 @@
+          HttpStatus.INTERNAL_SERVER_ERROR);
+      ex.printStackTrace();
+    }
+    return CompletableFuture.completedFuture(response);
+  }
+
+  /**
+   * Metodo che gestisce il servizio Feedback findByDottore.
+   */
+
+  @PostMapping(value = "/getByDottore", produces = MediaType.APPLICATION_JSON_VALUE)
+  public @ResponseBody CompletableFuture<ResponseEntity<RisultatoDto>> getNotificaByDottore(
+      @RequestBody DottoreDto input) throws Exception {
+    ResponseEntity<RisultatoDto> response;
+    try {
+      ListDto<NotificaDto> dto = getService().findByDottore(input.getCodiceFiscale());
+      response = new ResponseEntity<>(dto, HttpStatus.OK);
+    } catch (UserNotFoundException | AccessDeniedException ex) {
+      throw ex;
+    } catch (Exception ex) {
+      response = new ResponseEntity<>(new RisultatoDto("Errore durante la ricerca della notifica"),
+          HttpStatus.INTERNAL_SERVER_ERROR);
+      ex.printStackTrace();
+      ex.printStackTrace();
+    }
+    return CompletableFuture.completedFuture(response);
+  }
+
+  /**
+   * Metodo che gestisce il servizio Notifica getByTutore.
+   */
+  @PostMapping(value = "/getByTutore", produces = MediaType.APPLICATION_JSON_VALUE)
+  public @ResponseBody CompletableFuture<ResponseEntity<RisultatoDto>> getNotificaByTutore(
+      @RequestBody TutoreDto input) throws Exception {
+    ResponseEntity<RisultatoDto> response;
+    try {
+      ListDto<NotificaDto> dto = getService().findByDottore(input.getCodiceFiscale());
+      response = new ResponseEntity<>(dto, HttpStatus.OK);
+    } catch (UserNotFoundException | AccessDeniedException ex) {
+      throw ex;
+    } catch (Exception ex) {
+      response = new ResponseEntity<>(new RisultatoDto("Errore durante la ricerca della notifica"),
+          HttpStatus.INTERNAL_SERVER_ERROR);
+      ex.printStackTrace();
+      ex.printStackTrace();
+    }
+    return CompletableFuture.completedFuture(response);
+  }
+
+  /**
+   * Metodo che gestisce il servizio Notifica getByAdmin.
+   */
+  @PostMapping(value = "/getByAdmin", produces = MediaType.APPLICATION_JSON_VALUE)
+  public @ResponseBody CompletableFuture<ResponseEntity<RisultatoDto>> getNotificaByAdmin(
+      @RequestBody AdminDto input) throws Exception {
+    ResponseEntity<RisultatoDto> response;
+    try {
+      ListDto<NotificaDto> dto = getService().findByDottore(input.getCodiceFiscale());
+      response = new ResponseEntity<>(dto, HttpStatus.OK);
+    } catch (UserNotFoundException | AccessDeniedException ex) {
+      throw ex;
+    } catch (Exception ex) {
+      response = new ResponseEntity<>(new RisultatoDto("Errore durante la ricerca della notifica"),
+          HttpStatus.INTERNAL_SERVER_ERROR);
+      ex.printStackTrace();
+      ex.printStackTrace();
+    }
+    return CompletableFuture.completedFuture(response);
+  }
+
+  /**
+   * Metodo che gestisce il servizio save Notifica.
+   */
+
+  @PostMapping(value = "/save", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Wrong endpoint, this should be "/send", e.g.:
@PostMapping(value = "/send", produces = MediaType.APPLICATION_JSON_VALUE)